test/performance/odp_l2fwd.c

```diff
@@ -1661,6 +1680,40 @@ static void parse_args(int argc, char *argv[], appl_args_t *appl_args)
 		case 'g':
 			appl_args->num_groups = atoi(optarg);
 			break;
+		case 'I':
+			len = strlen(optarg);
+			if (len == 0) {
+				ODPH_ERR("Bad priority list\n");
+				exit(EXIT_FAILURE);
+			}
+			len += 1; /* add room for '\0' */
+
+			tmp_str = malloc(len);
+			if (tmp_str == NULL) {
+				ODPH_ERR("Priority list malloc() failed\n");
+				exit(EXIT_FAILURE);
+			}
+
+			strcpy(tmp_str, optarg);
```

Comment:
Static code checkers complain about usage of `strcpy()`.

Reply:
Fixed in v2. Changed all strcpy() to strncpy().